exams/Sum.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+using namespace std;
+int main() {
+    int i;
+    float Sum = 0, A[i];
+    cout<<"Enter size massive = "<<endl;

```

Comment:
```suggestion
    cout<<"Enter size array = "<<endl;
```
Better to rename "massive" to "array"

Reply:
Both names are not ok. Use semantic namings

Reply:
Я зробила відповідні виправлення до зауваження. 
#include <iostream>
using namespace std;
int main() {
    int i;
    float Sum = 0, A[i];
    cout<<"Enter the number of array elements = "<<endl;
    cin>>i;
    cout<<"Fill the array with numbers: "<<endl;
    for (int j = 0; j < i; j++){
        cin>>A[i];
        Sum+= A[i];
    }
    cout<<"Sum: "<<endl;
    cout<<Sum<<endl;

}

Reply:
> Both names are not ok. Use semantic namings

Я зробила відповідні виправлення до зауваження.
#include
using namespace std;
int main() {
int i;
float Sum = 0, A[i];
cout<<"Enter the number of array elements = "<<endl;
cin>>i;
cout<<"Fill the array with numbers: "<<endl;
for (int j = 0; j < i; j++){
cin>>A[i];
Sum+= A[i];
}
cout<<"Sum: "<<endl;
cout<<Sum<<endl;

}